.github/workflows/main.yml

```diff
@@ -34,7 +34,7 @@ env:
 
   # If you change this please also update GO_VERSION in Makefile (then run
   # `make lint` to see where else it needs to be updated as well).
-  GO_VERSION: 1.23.9
+  GO_VERSION: 1.23.10
```

Comment:
should we go to 1.24.5 straight away ?

Reply:
We normally remain backward compatible for 2 major versions. So we'd only go to 1.24 once 1.25 is out.